Fixed an upload bug when playbook contains regex starting with '=' character (#2396)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

## Unreleased

> * Fixed an issue where the **upload** command did not work when playbook contains regex starting with '=' character.

Reviewer: ```suggestion
* Fixed an issue where the **upload** failed on playbooks containing a value that starts with `=`.
```
